### PR TITLE
Team user not found in api request

### PR DESCRIPTION
**Pull request by rodrigovirgilio**
Rollbar: https://rollbar.com/cm-fulcrum/cm-fulcrum/items/319/

### CODE REVIEW

File: spec/api/v1/teams_spec.rb [hunk 1]
```diff
@@ -131,10 +131,27 @@
...
+    context 'when user team returns' do
+      it 'returns 2 team users' do
+        subject
 
-      expect(JSON.parse(response.body).count).to eq(2)
+        expect(JSON.parse(response.body).count).to eq(2)
+      end
+    end
+
+    context 'when no user team returns' do
```
Comment (andrerpbts) [inline spec/api/v1/teams_spec.rb]: `when team does not return any user`

File: spec/api/v1/teams_spec.rb [hunk 2]
```diff
@@ -131,10 +131,27 @@
 
     subject { get "/api/v1/teams/#{team.slug}/users", params: { per_page: 2, api_key: api_token.token } }
 
-    it 'returns 2 team users' do
-      subject
+    context 'when user team returns' do
```
Comment (andrerpbts) [inline spec/api/v1/teams_spec.rb]: `when team returns some users`

File: app/api/v1/teams.rb [hunk 3]
```diff
@@ -42,9 +42,13 @@ def authorize_team!
     paginate
     get '/:slug/users' do
       team = Team.includes(enrollments: [:user]).find_by(slug: params[:slug])
```
Comment (andrerpbts) [inline app/api/v1/teams.rb]: It's not being used anymore...

File: spec/api/v1/teams_spec.rb (hunk 1 above)
Comment (rodrigovirgilio) [reply spec/api/v1/teams_spec.rb]: because there is no team in the parameter (`params[:slug]`)

File: app/api/v1/teams.rb (hunk 3 above)
Comment (rodrigovirgilio) [reply app/api/v1/teams.rb]: It's true, I'll correct it.

File: spec/api/v1/teams_spec.rb [hunk 4]
```diff
@@ -131,10 +131,26 @@
...
+        expect(JSON.parse(response.body).count).to eq 2
+      end
+    end
+
+    context 'when team does not return any user' do
+      before do
+        team.users.destroy_all
 
-      expect(JSON.parse(response.body).count).to eq(2)
+        team.save
```
Comment (brunohkbx) [inline spec/api/v1/teams_spec.rb]: I think you can remove this line

Comment (rodrigovirgilio) [reply spec/api/v1/teams_spec.rb]: You are right, `destroy_all` will already remove the relationship of the users object with the team

**Pull request merged**